### PR TITLE
Added docstrings to print_all_arrays_in_memory

### PR DESCRIPTION
**Pull request by aryankhatana01**

### CODE REVIEW

**Comment by jkeane508** [comment]
Hi @aryankhatana01,

Can you please reference the issue related to this pull request :)

**Comment by aryankhatana01** [comment]
@jkeane508 Yeah, sorry, I forgot about that.
- [ ] #3476

File: ivy/functional/backends/torch/activations.py [hunk 1]
```diff
@@ -71,3 +71,15 @@ def softplus(x: torch.Tensor, /) -> torch.Tensor:
 
 
 softplus.unsupported_dtypes = ("float16",)
+
+
+def mish(
```
Comment (jkeane508) [inline ivy/functional/backends/torch/activations.py:76]: Is this part of the task?

File: ivy/functional/frontends/torch/non_linear_activation_functions.py [hunk 2]
```diff
@@ -34,3 +34,10 @@ def gelu(input, approximate="none"):
 
 
 gelu.unsupported_dtypes = ("float16",)
+
+
+def mish(input, inplace=False):
```
Comment (jkeane508) [inline ivy/functional/frontends/torch/non_linear_activation_functions.py:39]: Or this?

File: ivy/functional/ivy/activations.py [hunk 3]
```diff
@@ -359,3 +359,34 @@ def softplus(
 
     """
     return current_backend(x).softplus(x, out=out)
+
+
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+def mish(
```
Comment (jkeane508) [inline ivy/functional/ivy/activations.py:367]: Or this?

File: ivy/functional/ivy/general.py [hunk 4]
```diff
@@ -2333,7 +2333,23 @@ def num_arrays_in_memory():
 
 
 def print_all_arrays_in_memory():
-    """Prints all arrays which are currently alive."""
+    """
```
Comment (jkeane508) [inline ivy/functional/ivy/general.py:2336]: We should add an example documenting the behaviour of this function

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Just some questions regarding your changes to the files :)

File: ivy/functional/backends/torch/activations.py (hunk 1 above)
Comment (aryankhatana01) [reply ivy/functional/backends/torch/activations.py:76]: This was a mistake; I accidentally pushed to the wrong branch. I'll fix it in the next commit, and sure I'll add some examples :)
Thanks!

File: ivy/functional/ivy/general.py [hunk 5]
```diff
@@ -2333,7 +2333,30 @@ def num_arrays_in_memory():
...
+
+    Output
+    ------
+    Type and shape of all the Ivy native arrays which are currently alive.
+
+    Returns
+    -------
+    None
+
+    Example
```
Comment (jkeane508) [inline ivy/functional/ivy/general.py:2352]: I'm going to have to apologise, as this function returns framework specific class objects, we cannot implement and example in this case. If you could please remove that would be great

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Hi, sorry about this but Im actually going to have to ask you to remove the example. I didn't realise the type of function this was.

File: ivy/functional/ivy/general.py (hunk 5 above)
Comment (aryankhatana01) [reply ivy/functional/ivy/general.py:2352]: No problem at all! Gimme a minute.

Comment (aryankhatana01) [reply ivy/functional/ivy/general.py]: Is it okay now?

**Pull request merged**